src/promptflow/promptflow/_sdk/operations/_test_submitter.py

```diff
@@ -216,57 +204,57 @@ def change_logger_level(level):
             yield
             logger.setLevel(origin_level)
 
-        @contextlib.contextmanager
-        def add_prefix():
-            write = sys.stdout.write
-
-            def prefix_output(*args, **kwargs):
-                if args[0].strip():
-                    write(f"{Fore.LIGHTBLUE_EX}[{self.dataplane_flow.name}]: ")
-                write(*args, **kwargs)
-
-            sys.stdout.write = prefix_output
-            yield
-            sys.stdout.write = write
+        def show_node_log_and_output(node_run_infos, show_node_output):
+            """Show stdout and output of nodes."""
+            for node_name, node_result in node_run_infos.items():
+                # Prefix of node stdout is "%Y-%m-%dT%H:%M:%S%z"
+                pattern = r"\[\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{4}\] "
```

Comment:
It is not a good idea to depend on the output format here. If the format changes, this part will break.

Reply:
Yes, it is not a good idea to depend on the format of stdout. But the stdout of user code can only be obtained through the line result.
If the stdout format changed, it will not break print the output here. It will directly print the stdout